### DISCUSSION TITLE
What is |>

### DISCUSSION THREAD

**Discussion by formacion-tyris** [pointfreeco/swift-composable-architecture#1960] [General]
In Episode #13 The Many Faces of Map. 

func map<A, B>(_ f: @escaping (A) -> B) -> ([A]) -> [B] {
  return { xs in
	var result = [B]()
	xs.forEach {
		result.append(f($0))
	}
	return result
  }
}

**I found: [1, 2, 3] |> map(incr)**

**what is |> ?** 
why is not a . (dot) ?
is there another symbols combinations like this?

**And this? : >>>**

[1, 2, 3] |> map(incr) |> map(square)
[1, 2, 3] |> map(incr >>> square)

Thanks

**Comment by mbrandonw** [comment]
Hi @formacion-tyris, this is not the correct place to ask this question. Feel free to open a discussion on our "episode code samples" repo: https://github.com/pointfreeco/episode-code-samples/discussions
